mmdet3d/ops/norm.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, *args, **kwargs):
     def forward(self, input):
         assert input.dtype == torch.float32, \
             f'input should be in float32 type, got {input.dtype}'
-        if dist.get_world_size() == 1 or not self.training:
+        if not dist.is_initialized() or dist.get_world_size() == 1 or not self.training:
```

Comment:
Maybe we can add a comment indicating that `# if not in distributed training, we don't apply SyncBN`